AI-and-Analytics/Getting-Started-Samples/IntelPython_daal4py_GettingStarted/README.md

```diff
@@ -24,107 +24,130 @@ In this sample, you will run a batch Linear Regression model with oneDAL daal4py
 | Software                | Intel® oneAPI Data Analytics Library (oneDAL)
 > **Note**: AI and Analytics samples are validated on AI Tools Offline Installer. For the full list of validated platforms refer to [Platform Validation](https://github.com/oneapi-src/oneAPI-samples/tree/master?tab=readme-ov-file#platform-validation).
 
-### For Local Development Environments
+## Key Implementation Details
 
-You will need to download and install the following toolkits, tools, and components to use the sample.
+- This get started sample code is implemented for CPUs using the Python language. The example assumes you have daal4py and scikit-learn installed inside a conda environment, similar to what is delivered with the installation of the Intel® Distribution for Python* as part of the Intel(R) AI Tools.

```

Comment:
the same comment here - IDP is not a part of AI Tools

Reply:
updated